frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/pvc.scenario.ts

```diff
@@ -0,0 +1,86 @@
+import { browser } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import {
+  createNewPersistentVolumeClaim,
+  deletePersistentVolumeClaim,
+  goToPersistentVolumeClaims,
+  pvcStatus,
+} from '../../views/pvc.view';
+import {
+  NS,
+  PVC_STATUS,
+  SIZE_UNITS,
+  STORAGE_CLASS_PATTERNS,
+  VOLUME_ACCESS_MODES,
+} from '../../utils/consts';
+
+describe('Test PVC creation with options.', () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/`);
```

Comment:
It would be a bit safer to `await isLoaded()` here.

Reply:
isLoaded() function doesn't work here, because the page being loaded as default is Overview Dashboard, so I'd need dashboardIsLoaded()
But actually I wouldn't want the test to wait for the dashboard to be loaded, it might fail on https://bugzilla.redhat.com/show_bug.cgi?id=1753666 or just make the test run longer unnecessarily.
I do check that Persistent Volume Claims page is loaded at the beginning of each test as part of PVC creation function.

Reply:
ack